src/angular/date-input/date-input.ts

```diff
@@ -0,0 +1,149 @@
+import { Directive, ElementRef, Input, NgZone, Output, inject } from '@angular/core';
+import { booleanAttribute } from '@sbb-esta/lyne-angular/core';
+import type { SbbDateInputElement } from '@sbb-esta/lyne-elements/date-input.js';
+import { fromEvent, type Observable } from 'rxjs';
+import '@sbb-esta/lyne-elements/date-input.js';
+
+@Directive({
+  selector: 'sbb-date-input',
+})
+export class SbbDateInput<T = Date> {
```

Comment:
This class will need a ControlValueAccessor implementation, but it can be done in a follow-up PR.

Reply:
PR updated